gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -72,5 +94,42 @@ def start_postgres
     def register_standby_server
       run_repmgr_command(REGISTER_CMD)
     end
+
+    def configure_repmgrd
+      return true unless run_repmgrd_configuration
+      write_repmgrd_config
+      write_pgpass_file
+      start_repmgrd
```

Comment:
I would suggest returning `true` from `write_repmgrd_config` and update here to use the boolean `and` to only invoke the next method on success of the previous.

```ruby
def configure_repmgrd
  return true unless run_repmgrd_configuration
  write_repmgrd_config &&
  write_pgpass_file &&
  start_repmgrd
```